### ISSUE TITLE
Request to add release 0.0.6 to Ledger Live

### ISSUE THREAD

**Issue by riordant** [LedgerHQ/app-ssh-agent#27]
Current release is 0.0.6 but Ledger Live is 0.0.5, even with latest firmware on Nano S. Likely related to https://github.com/LedgerHQ/app-ssh-agent/issues/25

**Comment by sp4ke** [comment]
+1 would appreciate that or at least a build guide. 

I struggled to build it and had many errors which I managed to ignore by modifying a bit the Makefile.

I have now the app loaded with v0.0.6 however I'm getting the message "This app is not genuine" is there a way to get rid of it ?

**Comment by riordant** [comment]
Hi,
Did you get it to work? Could you share the make options? Thanks

**Comment by sp4ke** [comment]
I used a fresh latest ubuntu docker image, installed clang-6.0, libc6-dev-i386 and build-essentials. I did not use BLOS_ENV only BOLOS_SDK.

I change the following on the Makefile:

```
diff --git a/Makefile b/Makefile
index e20f209..b43552f 100755
--- a/Makefile
+++ b/Makefile
@@ -104,7 +104,7 @@ ifeq ($(GCCPATH),)
 $(info GCCPATH is not set: arm-none-eabi-* will be used from PATH)
 endif

-CC       := $(CLANGPATH)clang
+CC       := $(CLANGPATH)clang-6.0

 #CFLAGS   += -O0
 CFLAGS   += -O3 -Os
@@ -131,7 +131,7 @@ ifeq ($(TARGET_NAME),TARGET_NANOS)

         ifneq "$(wildcard $(BOLOS_SDK)/lib_ux/src/ux_flow_engine.c)" ""
                 SDK_SOURCE_PATH  += lib_ux
-                DEFINES                += HAVE_UX_FLOW
+#                DEFINES               += HAVE_UX_FLOW
         endif

 endif
```

**Comment by riordant** [comment]
And it works with these options? what about the command for `blue-loader-python`?

**Comment by sp4ke** [comment]
Yes it works and should load without problems using the `make load` command. If there are any issues with the load you can just manually provide the whole command line parameters to the loader. 

Using it on the ledger I did not notice anything broken yet other than the "Non genuine" message appearing everytime I launch the app.

**Comment by atishnazir** [comment]
Due #25, the latest Ledger Live release is 0.0.7, this issue can probably be closed.
